src/style.scss

```
@@ -64,8 +64,7 @@
 }
 
 [auro-icon] {
-  height: var(--ds-size-300, $ds-size-300);
-  width: var(--ds-size-300, $ds-size-300);
+  width: $ds-size-300;
```

Solution can not remove the token.

```suggestion
  width: var(--ds-size-300, $ds-size-300);
```

---

fixed